charts/application/templates/k8s-rolebinding.yaml

```diff
@@ -0,0 +1,36 @@
+{{- range .Values.serviceAccount.rbac.roleBindings }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
I'm not sure if it's possible, but can this `kind` also be set via a value?

Reply:
Shouldn't be a problem, might make the template a bit simpler.
Could do something like:
```yaml
rbac:
  - kind: RoleBinding
    roleType: ClusterRole
    roleName: admin
  - kind: ClusterRoleBinding
    roleType: ClusterRole
    roleName: admin
```

I'm waiting for Florian to make a decision if he wants the bindings in the templates in general

Reply:
if you validate for the possible values, go for it.

Reply:
Implemented as suggested above